tests/core/test_strtol.out

```diff
@@ -208,8 +208,6 @@ strtol("012", 0, 10) = 12
 strtol("0y12", 0, 0) = 0
 strtol("hello", 0, 30) = 14167554
 strtol("hello", 0, 10) = 0
-ERR Invalid argument
 strtol("not-a-number") = 0
-ERR Invalid argument
```

Comment:
why did this change?

Reply:
Oh, is this the EINVAL issue mentioned earlier?

Reply:
This is the change i mention in the description.     I also added a line about this to the comment.   The new behaviour is slightly different in this one edge case (where the very first char is bigger than base) but matches glibc so I think its actually an improvment.